subprojects/packagefiles/pcre/meson_options.txt

```diff
@@ -1,3 +1,5 @@
+option('shared_lib', type : 'boolean', value : false)
```

Comment:
why?

Reply:
Just reinstating what we had before.

Reply:
Ok, removed again, doing what @eli-schwartz suggested above.